src/components/Answers/Answers.tsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import './Answers.css';
 
-const Answers = () => {
+interface IProps {
+  answer: string | number
```

Comment:
We'll also be sending props of the question, and the country object, so we'll be sure to add those to IProps as well.